packages/web-pkg/src/composables/piniaStores/config/config.ts

```diff
@@ -29,7 +30,8 @@ const defaultOptions = {
   },
   openAppsInTab: false,
   routing: {
-    idBased: true
+    idBased: true,
+    fullShareOwnerPaths: false
```

Comment:
Is the default correct, there is no documentation about it anywhere

Reply:
It's optional and ocis doesn't set it, so it defaults to `false`. Your chosen default is correct :-)